### PR TITLE
README: fix typos and grammar

### PR DESCRIPTION
**Pull request by taivlam**
* Use more consistent capitalization
* Reword slightly some phrases/sentences to sound more natural
* Use symbolic link to refer to GH issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,41 +144,41 @@ None
...
-  To fix this, install libfuse2 as follows:
+  To fix this, install `libfuse2` as follows:
 
   ```
   sudo apt install libfuse2
   ```
 
 * On Ubuntu 24.04 and later, the AppImage will fail with a sandboxing error (`The SUID sandbox helper binary was found, but is not configured correctly...`)
-  This is due to AppArmor conflicting by restricting unprivileged user namespaces (https://github.com/jitsi/jitsi-meet-electron/issues/965,
+  This is due to an AppArmor conflict that restricts unprivileged user namespaces (jitsi/jitsi-meet-electron#965,
```
Comment (saghul) [inline README.md:164]: I think the other link syntax is much more widespread. Please undo the change.

Comment (taivlam) [reply README.md]: Ok, I've made the change.
